dotcom-rendering/src/components/LiveblogRightMultipleAdSlots.importable.tsx

```diff
@@ -4,27 +4,30 @@ import {
 	AD_CONTAINER_HEIGHT,
 	PADDING_BOTTOM,
 	SPACE_BETWEEN_ADS,
+	TOP_RIGHT_AD_STICKY_HEIGHT
 } from '../lib/liveblog-right-ad-constants';
+import { useAB } from '../lib/useAB';
 import { AdSlot } from './AdSlot';
 
-const calculateNumAdsThatFit = (rightColHeight: number) => {
+export const calculateNumAdsThatFit = (rightColHeight: number) => {
```

Comment:
Rejigged the logic since the right ad slot is outside of the liveblog-right ad region. There are tests for this now.